packages/mobile/src/providers/TokenUrl.tsx

```diff
@@ -0,0 +1,3 @@
+export const URL_TOKEN: string | null = null;
```

Comment:
Would this be the file that we write the actual urls to?

Reply:
Yes. will provide details